Community Contributors newspost (#2068)
ppy/osu-wiki: news/2019-02-22-acknowledging-community-contributors.md

---
layout: post
title: "Acknowledging Community Contributors"
date: 2019-02-22 00:00:00 +0000
---

At its very heart, osu! is almost entirely driven by its community. And at the heart of our community, the efforts of an outstanding few make themselves known. Today, we take a moment to reflect on some of these people, and award them the recognition they deserve.

![](/wiki/shared/news/2019-02-22-acknowledging-community-contributors/banner.jpg)

> Perhaps the most prestigious of all profile badges awarded, the [community contributor badge](https://assets.ppy.sh/profile-badges/contributor.png) is expressly granted to those of the osu! community who have gone utterly above and beyond the norm in their enduring contribution to the well-being of the game and its community.

Reviewer: Is the badge supposed to be this one? https://assets.ppy.sh/profile-badges/contributor.jpg

Author: Good catch!

> In a first for many years, we are proud to announce the latest set of community contributors in what is the largest number of additions made since the initiative started back in 2014.

Reviewer 2: Did research. [Wiki](https://osu.ppy.sh/help/wiki/People/Community_Contributors#2013) | [News](https://osu.ppy.sh/home/news/2013-08-08-osu-community-contributors)
```suggestion
In a first for many years, we are proud to announce the latest set of community contributors in what is the largest number of additions made since the initiative started back in 2013.
```

> Please note however - this list is by no means exhaustive, and only represents the acknowledgement of continued, consistent contribution across a long period of time as observed and noted by the osu!team.

Reviewer 3: ```suggestion
Please note, however: this list is by no means exhaustive, and only represents the acknowledgement of continued, consistent contribution across a long period of time as observed and noted by the osu!team.
```

## Community

### HappyStick

> [![](/wiki/shared/news/2019-02-22-acknowledging-community-contributors/people/HappyStick.jpg)](https://osu.ppy.sh/u/256802)

Reviewer 3: use `/users` links (unless if there was a specific reason to use short ones here?)

Reviewer 2: `/u` is the old site.

All but leaving behind a successful streaming career and tournament series of his own, HappyStick's jaunt into the deeper affairs of the community has proven instrumental in the continuation of the osu! World Cup series in the wake of its previous organizer's farewell.

Beyond his award-winning smile, HappyStick has proven himself to be an invaluable addition to the osu!team, heading the new osu! Coffee Hour series, generally spending many sleepless nights setting up and streaming World Cup matches, creating tools to ensure it is easier for others to run them, and so much more.

We could say many things about HappyStick, but foremost among them is how thankful we are for his continued contribution.

### juankristal

[![](/wiki/shared/news/2019-02-22-acknowledging-community-contributors/people/juankristal.jpg)](https://osu.ppy.sh/u/443656)

Something of a living meme among the osu! tournament scene, juankristal has been perpetually blamed for everything and anything that has gone both wrong and right for countless years.

> For once, we are delighted to firmly pin the blame on him for the continuation of the osu! World Cup series as a whole, stepping up to lead the INTENSE organization required to schedule, oversee and generally maintain multiple multi-national tournaments. 

Reviewer 3: ```suggestion
For once, we are delighted to firmly pin the blame on him for the continuation of the osu! World Cup series as a whole, stepping up to lead the INTENSE organization required to schedule, oversee and generally maintain multiple multinational tournaments.
```

The workload involved in running the osu! World Cups is *immense*, perhaps far greater than most people realize, and juankristal has not only stepped up himself to do it out of a sheer desire to see it done, but has *excelled* at it, allowing the series to continue without so much as a blip.

### WalterToro

[![](/wiki/shared/news/2019-02-22-acknowledging-community-contributors/people/WalterToro.jpg)](https://osu.ppy.sh/u/5281416)

A mainstay force within the Global Moderation Team, WalterToro is not your ordinary chat moderator. He is *everywhere*, in the osu!wiki scene, in the tournaments scene, and wildly active in the Spanish community to say the least.

From refereeing matches to refereeing people, we are intensely thankful for WalterToro's incredible work ethic and tireless drive to do what needs to be done.

### clayton

> [![](/wiki/shared/news/2019-02-22-acknowledging-community-contributors/people/clayton.jpg)](https://osu.ppy.sh/u/3666350)

Reviewer 3: more like I'm too uninspired to choose an avatar, but I'll take it B)

thanks bird man & co. <3

Getting things done is what keeps osu! alive and moving, and there is perhaps no better example of this than clayton in recent times. We are supremely thankful for his efforts over the past few years.

### VeilStar

[![](/wiki/shared/news/2019-02-22-acknowledging-community-contributors/people/VeilStar.jpg)](https://osu.ppy.sh/u/4255720)

A formerly gigantic presence in the forum technical support team, VeilStar has been responsible for the resolution of literally *thousands* of technical issues over the years and has helped similar amounts of people enjoy osu! to its fullest.

Having since departed the team and osu! in general to pursue things elsewhere, we'd like to commit just how thankful we are for his assistance over the years.

### Stefan

[![](/wiki/shared/news/2019-02-22-acknowledging-community-contributors/people/Stefan.jpg)](https://osu.ppy.sh/u/626907)

> The very picture of an unsung hero, Stefan has single-handedly maintained the [beatmap packs](https://osu.ppy.sh/beatmaps/packs) by himself for years regularly and consistently, completely without prompting and entirely out of a drive to keep the project alive. 

Reviewer 3: ```suggestion
The very picture of an unsung hero, Stefan has single-handedly maintained the [beatmap packs](https://osu.ppy.sh/beatmaps/packs) by himself for years regularly and consistently, completely without prompting and entirely out of a drive to keep the project alive.
```
just the whitespace at the end, if that's hard to see in diff

Through thick and thin, Kurokami's ceaseless direction of the project has kept it alive where circumstance would have doomed it to be forgotten, and it is this sheer determination and drive to see something done that we are tremendously grateful to benefit from.

### p3n

[![](/wiki/shared/news/2019-02-22-acknowledging-community-contributors/people/p3n.jpg)](https://osu.ppy.sh/u/123703)

Our initial reaction to p3n's name being raised was universally: "how does he not have this badge already?"

So vast and numerous are p3n's contributions over the years - from creating a major metric used by the Quality Assurance Team to weigh and compare the activity of a modder in a given period of time, to entire written drafts and improvements on the basic modding/ranking system, his previous and enduring leadership of both the Beatmap Approval Team and the current QAT, assistance with the osu! World Cups and so, so much more.

### shARPII

[![](/wiki/shared/news/2019-02-22-acknowledging-community-contributors/people/shARPII.jpg)](https://osu.ppy.sh/u/776257)

> Departed from the osu!team earlier in the year, shARPII was once the primary manager of the Global Moderation Team. Not merely content with one herculean task, shARPII also decided he'd write a utility to help the osu! World Cup staff handle the arduous process of handling multiplayer games from a referee's perspective, coding the program who is now affectionally dubbed *Script-chan* by the tournaments crew.

Reviewer 2: ```suggestion
Departed from the osu!team earlier in the year, shARPII was once the primary manager of the Global Moderation Team. Not merely content with one herculean task, shARPII also decided he'd write a utility to help the osu! World Cup staff handle the arduous process of handling multiplayer games from a referee's perspective, coding the program who is now affectionately dubbed *Script-chan* by the tournaments crew.
```

shARPII's passion and drive to improve upon and iterate some of the oldest remaining fixtures of the game's volunteer groups led to substantial improvements made for everybody, and we really cannot thank him enough for his time with us.

## Project Loved

Formed from the ashes of the old Loved Captain's Pick system pioneered over a year ago, Project Loved has continually evolved and improved upon itself, becoming almost a secondary content stream from the efforts of its organizers, week in, week out.

Few community-run projects have sustained such momentum and attained such reach in all of osu!'s history, and it is only fitting that the first batch of new community contributors for 2018 originate from the ranks of this much beloved project.

### Toy

[![](/wiki/shared/news/2019-02-22-acknowledging-community-contributors/people/Toy.jpg)](https://osu.ppy.sh/u/2757689)

The primary drive and chief organizer of Project Loved, Toy's efforts to revitalize and shape the Loved beatmap category are at the very heart of the program, and it is very safe to say that without him, it would simply not exist.

> Beyond this, his contributions towards moderating the defacto osu! subreddit and his endless encouragement and positive attitude are shining examples of the things that make the osu! community as special as it is, and it is with great delight that we welcome him into the midst of the community contributors with his *20th* profile badge to boot.

Reviewer 4: it's actually my 22nd badge eeeeeeeeeeeeea

> Beyond this, his contributions towards moderating the defacto osu! subreddit and his endless encouragement and positive attitude are shining examples of the things that make the osu! community as special as it is, and it is with great delight that we welcome him into the midst of the community contributors with his *20th* profile badge to boot.

Reviewer 4: it's actually my 22nd badge eeeeeea

Yes, you read that right.

### Kaifin

[![](/wiki/shared/news/2019-02-22-acknowledging-community-contributors/people/Kaifin.jpg)](https://osu.ppy.sh/u/2596942)

Never far from the fray, Kaifin's continued assistance in the day-to-day affairs of Project Loved have had an immeasurable impact on its consistency and persistence overall.

> The longevity of an project is built upon the backs of people like Kaifin, and we are very grateful for his continued support.

Reviewer 3: ```suggestion
The longevity of a project is built upon the backs of people like Kaifin, and we are very grateful for his continued support.
```

### Halfslashed

[![](/wiki/shared/news/2019-02-22-acknowledging-community-contributors/people/Halfslashed.jpg)](https://osu.ppy.sh/u/4598899)

Very much the forward 'face' of the project, Halfslashed has been deep in the thick of the social aspects of leadership, being largely responsible for the deluge of speedmapping and other mapping contests that form the bulk of the competitive scene within the program.

### Mir

[![](/wiki/shared/news/2019-02-22-acknowledging-community-contributors/people/Mir.jpg)](https://osu.ppy.sh/u/8688812)

An effortless generalist and a former mentee of the program himself under Shiirn, Mir's presence in the project was confined to no one particular domain, but showcased his ability to coordinate and implement at a level that would later speak well in his favor for his entry into the Quality Assurance Team.

### Mun

[![](/wiki/shared/news/2019-02-22-acknowledging-community-contributors/people/Mun.jpg)](https://osu.ppy.sh/u/6699165)

Once a former mentee himself under Shiirn, Mun's efforts have spanned across dozens of domains within the project, primarily handling moderation and evaluation related tasks pertaining to both mentors and mentees.

> ### J1nx1337

Reviewer 5: Very minor correction on the spelling. :p
Thank you so much! ^^
```suggestion
### J1NX1337
```

[![](/wiki/shared/news/2019-02-22-acknowledging-community-contributors/people/J1nx1337.jpg)](https://osu.ppy.sh/u/3971179)

> Responsible for handling much of the busywork deluge that often accompanies managing a project of such size and consistency, J1nx1337's efforts were instrumental to the continuation of the project, and while not all work is exactly glamorous, such things are certainly no less important towards making things happen.

Reviewer 5: ```suggestion
Responsible for handling much of the busywork deluge that often accompanies managing a project of such size and consistency, J1NX1337's efforts were instrumental to the continuation of the project, and while not all work is exactly glamorous, such things are certainly no less important towards making things happen.
```

-------------

As mentioned in previous years, we cannot really stress just how important community contribution is to the future of osu! as a whole. 

Unlike many other game projects on the web, much of what drives osu! originates almost wholly from its community, meaning that individual people can have a **massive** influence in making the game awesome for everybody else.

And we hope to see the trend continue well into our second decade of clicking circles, beating drums, catching fruit and smashing keys!

> —Ephemeral

Reviewer 3: ```suggestion
—Ephemeral and the osu!team
```
since "we" is used a lot throughout this post

Author: this should probably just be osu!team

Reviewer 3: edited

---
layout: post
title: "Acknowledging Community Contributors"
date: 2019-02-22 00:00:00 +0000
---

At its very heart, osu! is almost entirely driven by its community. And at the heart of our community, the efforts of an outstanding few make themselves known. Today, we take a moment to reflect on some of these people, and award them the recognition they deserve.

![](/wiki/shared/news/2019-02-22-acknowledging-community-contributors/banner.jpg)

Perhaps the most prestigious of all profile badges awarded, the [community contributor badge](https://assets.ppy.sh/profile-badges/contributor.jpg) is expressly granted to those of the osu! community who have gone utterly above and beyond the norm in their enduring contribution to the well-being of the game and its community.

> In a first for many years, we are proud to announce the latest set of community contributors in what is the largest number of additions made since the initiative started back in 2013.

Reviewer 6: This almost sounds like we're announcing community contributors for the first time in many years as opposed to us announcing its the largest number of additions in many years. Perhaps omitting the first part is a good idea.

Author: i fix

![](/wiki/shared/news/2019-02-22-acknowledging-community-contributors/banner.jpg)

Perhaps the most prestigious of all profile badges awarded, the [community contributor badge](https://assets.ppy.sh/profile-badges/contributor.jpg) is expressly granted to those of the osu! community who have gone utterly above and beyond the norm in their enduring contribution to the well-being of the game and its community.

We are proud to announce the latest set of community contributors in what is the largest number of additions made since the initiative started back in 2013.

Please note, however: this list is by no means exhaustive, and only represents the acknowledgement of continued, consistent contribution across a long period of time as observed and noted by the osu!team.

## Development

### ekr

[![](/wiki/shared/news/2019-02-22-acknowledging-community-contributors/people/ekr.jpg)](https://osu.ppy.sh/users/4497706)

Responsible for considerable contributions to the osu!lazer project, ekr's efforts have focused foremost on reviewing code and making grand steps towards development of the game's input system, essentially rewriting it from scratch.

> Noted specifically as one of the few developers brave enough to delve into the core architecture of osu!'s framework, ekr's work ethic and willingness to get involved is highly valued.

Author: by us, clearly

## Community

### HappyStick

[![](/wiki/shared/news/2019-02-22-acknowledging-community-contributors/people/HappyStick.jpg)](https://osu.ppy.sh/users/256802)

All but leaving behind a successful streaming career and tournament series of his own, HappyStick's jaunt into the deeper affairs of the community has proven instrumental in the continuation of the osu! World Cup series in the wake of its previous organizer's farewell.

> Beyond his award-winning smile, HappyStick has proven himself to be an invaluable addition to the osu!team, heading the new osu! Coffee Hour series, generally spending many sleepless nights setting up and streaming World Cup matches, creating tools to ensure it is easier for others to run them, and so much more.

Reviewer 6: ```suggestion
Beyond his award-winning smile, HappyStick has proven himself to be an invaluable addition to the osu!team, heading the new osu! Coffee Hour series, generally spending many sleepless nights setting up and streaming World Cup matches, creating tools to ensure that it is easier for others to run them, and so much more.
```

> We could say many things about HappyStick, but foremost among them is how thankful we are for his continued contribution.

Reviewer 6: ```suggestion
We could say many more things about HappyStick, but foremost among them is how thankful we are for his continued contribution.
```
considering we've already said many things

### juankristal

[![](/wiki/shared/news/2019-02-22-acknowledging-community-contributors/people/juankristal.jpg)](https://osu.ppy.sh/users/443656)

Something of a living meme among the osu! tournament scene, juankristal has been perpetually blamed for everything and anything that has gone both wrong and right for countless years.

For once, we are delighted to firmly pin the blame on him for the continuation of the osu! World Cup series as a whole, stepping up to lead the INTENSE organization required to schedule, oversee and generally maintain multiple multinational tournaments.

> The workload involved in running the osu! World Cups is *immense*, perhaps far greater than most people realize, and juankristal has not only stepped up himself to do it out of a sheer desire to see it done, but has *excelled* at it, allowing the series to continue without so much as a blip.

Reviewer 6: ```suggestion
The workload involved in running the osu! World Cups is *immense*, perhaps far greater than most people realize, and juankristal has not only stepped himself up to do it out of a sheer desire to see it done, but has *excelled* at it, allowing the series to be continued without so much as a blip.
```
considering you've been using past tense up to this point

### WalterToro

[![](/wiki/shared/news/2019-02-22-acknowledging-community-contributors/people/WalterToro.jpg)](https://osu.ppy.sh/users/5281416)

> A mainstay force within the Global Moderation Team, WalterToro is not your ordinary chat moderator. He is *everywhere*, in the osu!wiki scene, in the tournaments scene, and wildly active in the Spanish community to say the least.

Reviewer 6: semicolon after everywhere?

Reviewer 3: should be colon

> From refereeing matches to refereeing people, we are intensely thankful for WalterToro's incredible work ethic and tireless drive to do what needs to be done.

Reviewer 6: intensely thankful seems like a rather odd way to be thankful imo

Author: altered

### clayton

[![](/wiki/shared/news/2019-02-22-acknowledging-community-contributors/people/clayton.jpg)](https://osu.ppy.sh/users/3666350)

From his involvement in the development of the new website, to reviewing countless improvements and additions to the osu!wiki and various newsposts, to assisting with the maintenance and general day-to-day tasks with Project Loved and much more, clayton has done almost everything there is to do, and continues to keep doing it.

Getting things done is what keeps osu! alive and moving, and there is perhaps no better example of this than clayton in recent times. We are supremely thankful for his efforts over the past few years.

### VeilStar

[![](/wiki/shared/news/2019-02-22-acknowledging-community-contributors/people/VeilStar.jpg)](https://osu.ppy.sh/users/4255720)

> A formerly gigantic presence in the forum technical support team, VeilStar has been responsible for the resolution of literally *thousands* of technical issues over the years and has helped similar amounts of people enjoy osu! to its fullest.

Reviewer 6: ```suggestion
A formerly gigantic presence in the forums' technical support team, VeilStar has been responsible for the resolution of literally *thousands* of technical issues over the years and has helped similar amounts of people enjoy osu! to its fullest.
```
Considering forums are generally plural

Author: reads completely wrong to me even if it's probably (maybe) grammatically correct

Reviewer 3: writing it as a possessive like this implies that each forum has its own technical support team, which isn't true

```suggestion
A formerly gigantic presence in the technical support team of the forums, VeilStar has been responsible for the resolution of literally *thousands* of technical issues over the years and has helped similar amounts of people enjoy osu! to its fullest.
```

maybe like this?

Reviewer 6: Is that not possessive as well? I mean it does read better so I wouldn't mind this instead.

Author: i'll meet inbetween and omit any mention of the forums

> Having since departed the team and osu! in general to pursue things elsewhere, we'd like to commit just how thankful we are for his assistance over the years.

Reviewer 7: Usage of `commit` is awkward in this sentence. `show`, `express`, or similar would work better.

Author: fixed

### Stefan

> [![](/wiki/shared/news/2019-02-22-acknowledging-community-contributors/people/Stefan.jpg)](https://osu.ppy.sh/users/626907)

Reviewer 7: Current avatar is different than one pictured in banner. Not sure if it matters at all but pointing it out anyways.

Author: oh my god he's changed it since yesterday AHHHHHH

> The very picture of an unsung hero, Stefan has single-handedly maintained the [beatmap packs](https://osu.ppy.sh/beatmaps/packs) by himself for years regularly and consistently, completely without prompting and entirely out of a drive to keep the project alive.

Reviewer 6: ```suggestion
The very picture of an unsung hero, Stefan has single-handedly maintained the [beatmap packs](https://osu.ppy.sh/beatmaps/packs) by himself for years regularly and consistently, completely without being prompted and entirely out of a drive to keep the project alive.
```

His efforts have not gone unnoticed.

### Kurokami

[![](/wiki/shared/news/2019-02-22-acknowledging-community-contributors/people/Kurokami.jpg)](https://osu.ppy.sh/users/260933)

A long term member of the Beatmap Nominators, then the Quality Assurance Team, Kurokami has been a regular presence in beatmapping-related affairs for literal years. His most notable contribution is his leadership of the Beatmap Spotlights project, and its continued effort to bring only the highest quality of beatmaps to bear for the community's appraisal.

Through thick and thin, Kurokami's ceaseless direction of the project has kept it alive where circumstance would have doomed it to be forgotten, and it is this sheer determination and drive to see something done that we are tremendously grateful to benefit from.

### p3n

[![](/wiki/shared/news/2019-02-22-acknowledging-community-contributors/people/p3n.jpg)](https://osu.ppy.sh/users/123703)

Our initial reaction to p3n's name being raised was universally: "how does he not have this badge already?"

> So vast and numerous are p3n's contributions over the years - from creating a major metric used by the Quality Assurance Team to weigh and compare the activity of a modder in a given period of time, to entire written drafts and improvements on the basic modding/ranking system, his previous and enduring leadership of both the Beatmap Approval Team and the current QAT, assistance with the osu! World Cups and so, so much more.

Reviewer 6: this is a REALLY long sentence. Can we somehow split this list up with a full stop?

Author: it's intentionally phrased this way to highlight the immense breadth of his contributions

### shARPII

[![](/wiki/shared/news/2019-02-22-acknowledging-community-contributors/people/shARPII.jpg)](https://osu.ppy.sh/users/776257)

> Departed from the osu!team earlier in the year, shARPII was once the primary manager of the Global Moderation Team. Not merely content with one herculean task, shARPII also decided he'd write a utility to help the osu! World Cup staff handle the arduous process of handling multiplayer games from a referee's perspective, coding the program who is now affectionately dubbed *Script-chan* by the tournaments crew.

Reviewer 7: shARPII retired in 2018. Also, in the last part, `who` -> `which` is more correct imo but I think this is pretty minor

Author: fixed

Very much the forward 'face' of the project, Halfslashed has been deep in the thick of the social aspects of leadership, being largely responsible for the deluge of speedmapping and other mapping contests that form the bulk of the competitive scene within the program.

### Mir

[![](/wiki/shared/news/2019-02-22-acknowledging-community-contributors/people/Mir.jpg)](https://osu.ppy.sh/users/8688812)

An effortless generalist and a former mentee of the program himself under Shiirn, Mir's presence in the project was confined to no one particular domain, but showcased his ability to coordinate and implement at a level that would later speak well in his favor for his entry into the Quality Assurance Team.

### Mun

[![](/wiki/shared/news/2019-02-22-acknowledging-community-contributors/people/Mun.jpg)](https://osu.ppy.sh/users/6699165)

Once a former mentee himself under Shiirn, Mun's efforts have spanned across dozens of domains within the project, primarily handling moderation and evaluation related tasks pertaining to both mentors and mentees.

### J1NX1337

[![](/wiki/shared/news/2019-02-22-acknowledging-community-contributors/people/J1nx1337.jpg)](https://osu.ppy.sh/users/3971179)

> Responsible for handling much of the busywork deluge that often accompanies managing a project of such size and consistency, J1NX1337's efforts were instrumental to the continuation of the project, and while not all work is exactly glamorous, such things are certainly no less important towards making things happen.

Reviewer 7: Fairly minor but `deluge` is used for the second time (in the Mentorship section alone). Just removing it here should be fine

Author: also fixed

-------------

As mentioned in previous years, we cannot really stress just how important community contribution is to the future of osu! as a whole. 

Unlike many other game projects on the web, much of what drives osu! originates almost wholly from its community, meaning that individual people can have a **massive** influence in making the game awesome for everybody else.

> And we hope to see the trend continue well into our second decade of clicking circles, beating drums, catching fruit and smashing keys!

Reviewer 6: This is just me, but having a line break and then starting with an and can potentially cause the reader to lose the subject.

Author: resolved
